package.json

```diff
@@ -135,6 +137,7 @@
     "jest": "^25.1.0",
     "jest-puppeteer": "^4.4.0",
     "puppeteer": "^2.1.1",
+    "start-server-and-test": "^1.10.11",
```

Comment:
used the same trick as #1058, which has been working nicely (I've had the smoke tests pass in actions 10 or so times now, on my testing repo).